src/models/Scholarships.test.ts

```diff
@@ -90,15 +93,19 @@ test('converter.fromFirestore', () => {
     description: 'description',
     deadline: firebase.firestore.Timestamp.fromDate(new Date('2019-02-20')),
```

Comment:
You could extract these `new Date(...)` variables and use them instead. That way we don't get multiple dates mixed up :)